documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    DOWNLOAD_LOCATION = "/root/Downloads/"
</pre><p>Any reason for using /root/ for temporary storage? This works in colab but wouldn't work if a user downloaded the notebook and ran jupyter locally.</p><p> </p><p>Even in Colab, using the directory you start in is good practice, as it's the default dir in the files side-panel, so it's much easier for users to interact with the file system.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>